common.yml

```diff
@@ -7,9 +7,9 @@ command_type: system
 # run the same task in the previous revision if the current task fails
 stepback: true
 
-mongo_tools_variables:
+variables:
```

Comment:
This should've been `variables` since `mongo_tools_variables` isn't a valid keyword to define on the root.

See [this line](https://github.com/evergreen-ci/evergreen/blob/a4d568801804ccc151e65fcaaaa00e3886be9fbd/self-tests.yml#L40) of Evergreen's config file.